dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/MethodSymbol.java

```diff
@@ -94,4 +94,11 @@ public String toString() {
         + "'s method "
         + signature;
   }
+
+  /** Returns the method name with its signature. */
+  public String getMethodNameWithSignature() {
```

Comment:
toString would print the class name too, which was unnecessary for showing AbstractMethodProblem's toString.

Reply:
Fine, though it probably doesn't need to be public

Reply:
Good catch. Removed public.